src/test/functional/test_test.ts

```diff
@@ -1,34 +1,21 @@
 Feature('testing');
```

Comment:
Is this test file test_test.ts still required since u already have a proper functional test for signing in. Please remove the file if not required anymore

Reply:
Removed the file